### DISCUSSION TITLE
Invalid nodejs download link

### DISCUSSION THREAD

**Discussion by juno-junho** [artilleryio/artillery#2375] [Artillery]
![image](https://github.com/artilleryio/artillery/assets/72647031/0d56fbe7-142c-4ebc-88f3-497fc954ba59)
Hyperlink to Node.js, at System requirements category, needs to be updated.
node js download link is no longer available.

shoud change hyperlink `https://nodejs.org/en/about/releases/ ` to `https://nodejs.org/en/download`

**Comment by bernardobridge** [comment]
Hey @juno-junho , thanks for reporting this 🙇 . It's been fixed!
